src/lib/tpm.c

```diff
@@ -3444,7 +3444,7 @@ static EC_POINT *tpm_pub_to_ossl_pub(EC_GROUP *group, TPM2B_PUBLIC *key) {
         goto out;
     }
 
-    int rc = EC_POINT_set_affine_coordinates_GFp(group,
+    int rc = EC_POINT_set_affine_coordinates(group,
```

Comment:
Please note that EC_POINT_set_affine_coordinates has been added in OpenSSL 1.1.1, see https://www.openssl.org/docs/manmaster/man3/EC_POINT_set_affine_coordinates.html.
If we require OpenSSL >= 1.1.0, we have to live with EC_POINT_set_affine_coordinates_GFp.